src/components/Section/Foreign/Passport/Passport.jsx

```diff
@@ -311,7 +283,6 @@ Passport.defaultProps = {
   HasPassports: {},
   suggestedNames: [],
   reBook: '^[a-zA-Z]{1}[0-9]{6,9}$',
```

Comment:
I happened to have my passport near by and it has a type letter and then a 9 digit passport number.  When I entered that schema into the form, I cannot input all 9 digits into the form. Looks like the regex only allows 9 places including the letter. Should this be updated to `7-10` to allow the initial letter space?

Reply:
Looks like the regex is fine. Just need to make the max length of `10`

Reply:
The help text is also a little misleading as well, the passport number in my book doesn't technically begin with a letter... I guess that is referring to the `Type` letter?

Reply:
Should we turn it into two inputs? One is a letter type and the other is a 6-9 digit number?

Reply:
This seems inaccurate as well `Passports will start with the letter A.` https://github.com/18F/e-QIP-prototype/pull/839/files#diff-f3e54c23a039ce2474f26a85429c3bc8R79, @ryanhofdotgov do you know the source of that help text? The paper SF-86 makes no mention of needing a letter at all.